testsuite/features/secondary/min_salt_migration.feature

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+# This feature depends on 
+# - minkvm_guests.feature which has already configured the KVM Minion to be a virtualization host.
+# - srv_sync_channels.feature which syncs SLES 15 SP4 and the client tools.
+# We also test 'Bootstrapping using the command line' in this feature
+
+@scope_salt
+@scope_salt_ssh
+@scope_virtualization
+@virthost_kvm
+Feature: Migrate Salt to bundled Salt on a nested Minion VM
+
+  Scenario: Log in as admin user in the Salt migration context
+    Given I am authorized for the "Admin" section
+
+  Scenario: Show the KVM host system overview
+    Given I am on the Systems overview page of this "kvm_server"
+
+  Scenario: Prepare a SLES KVM test VM and list it in the Salt migration context
```

Comment:
I would have these two scenarios combined.

Reply:
Makes sense. This was copy pasting from the current KVM tests. I will merge it.

Reply:
Fixed.